portfolio/src/main/java/com/google/sps/servlets/LoadForm.java

```diff
@@ -0,0 +1,64 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.cloud.datastore.Datastore;
+import com.google.cloud.datastore.DatastoreOptions;
+import com.google.cloud.datastore.Entity;
+import com.google.cloud.datastore.Query;
+import com.google.cloud.datastore.QueryResults;
+import com.google.cloud.datastore.StructuredQuery.OrderBy;
+import com.google.gson.Gson;
+import com.google.protobuf.Message;
+
+import org.jsoup.Jsoup;
+import org.jsoup.safety.Whitelist;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet responsible for listing tasks. */
+@WebServlet("/list-messages")
+public class LoadForm extends HttpServlet {
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+   
+    Datastore datastore = DatastoreOptions.getDefaultInstance().getService();
+    Query<Entity> query =
+        Query.newEntityQueryBuilder().setKind("Task").setOrderBy(OrderBy.desc("name")).build();
+    QueryResults<Entity> results = datastore.run(query);
+
+    List<String []> messages = new ArrayList<>();
+    while (results.hasNext()) {
+      Entity entity = results.next();
+      String name = entity.getString("name");
+      String email = entity.getString("email");
+      String message = entity.getString("message");
+      String [] content ={"Message ==>"+"Name:"+name, "||"+ "Email:"+ email,"||"+"Message:"+ message +
```

Comment:
This is optional, but in general, a good idea.

Consider returning just the data, instead of a formatted block and doing the formatting on the JS side.

```
String content = String.format(`{"name": "%s", "message": "%s"}`, name, message);
```
[String.format() documentation](https://www.javatpoint.com/java-string-format)

Then on the Javascript side you can fetch the values by doing something like, and format the message how you would like there.
```
resp[i]["message"]
```

The reason for this is that it's faster to handle UI/formatting changes through the JS side because it runs on the "client" side of the "client-server" relationship. You can quickly push a change to the JS side by just updating the JS file. I believe App Engine local development can even live update the JS file without running any `gcloud` commands.

A change to the Java side requires re-deploying and restarting your server.